cnd/src/storage.rs

```diff
@@ -1039,57 +1045,6 @@ impl
     }
 }
 
-#[async_trait::async_trait]
-impl Load<SwapContext> for Storage {
-    async fn load(&self, swap_id: LocalSwapId) -> anyhow::Result<SwapContext> {
-        #[derive(QueryableByName)]
-        struct Result {
-            #[sql_type = "sql_types::Text"]
-            role: Text<Role>,
-            #[sql_type = "sql_types::Text"]
-            alpha_protocol: Text<Protocol>,
-            #[sql_type = "sql_types::Text"]
-            beta_protocol: Text<Protocol>,
-        }
-
-        let Result { role, alpha_protocol, beta_protocol } = self.db.do_in_transaction(|connection| {
-            // Here is how this works:
```

Comment:
Is this comment preserved in the other impl that we are now using?

Reply:
Good question! It is :)